src/ResourceManager/Automation/Commands.Automation.Test/ScenarioTests/AutomationTests.ps1

```diff
@@ -361,3 +361,62 @@ function Test-AutomationStartUnpublishedRunbook
     Remove-AzureRmAutomationRunbook $accountName -Name $runbook.Name -Force 
     Assert-Throws {Get-AzureRmAutomationRunbook $accountName -Name $runbook.Name -Parameters $runbookParameters -PassThru -ErrorAction Stop}
 }
+
+<#
+.SYNOPSIS
+Tests create new automation variable with Get-Content
+#>
+function Test-CreateNewVariableWithGetContent
+{
+	$resourceGroupName = Get-RandomResourceGroupName
+	$rgLocation = "japaneast"
+	$automationAccountName = Get-RandomAutomationAccountName
+    $automationAccountLocation = $rgLocation
+
+	$variableName = "GetContentVariable"
+
+	$fileName = ".\CreateNewVariableWithGetContent.txt"
```

Comment:
This will not work with tests running in parallel - the current directory may be changed by a concurrently executing test.  You can get the current directory using this pattern: https://github.com/Azure/azure-powershell/blob/bbe5ee957de07032bcf7f9062bba91cceee62565/src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/DiagnosticsExtensionTests.cs#L26

Reply:
I will fix this

Reply:
We will use  $env:temp\CreateNewVariableWithGetContent.txt